From 681c7c313dbf918948e119152cf6acd0e548304a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E6=84=9B=E8=8E=89=E5=B8=8C=E9=9B=85?= <1138745761@qq.com>
Date: Wed, 3 Dec 2025 10:33:33 +0800
Subject: [PATCH] patch

---
 fs/namespace.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/fs/namespace.c b/fs/namespace.c
index 95857699b90d..e92240bf17ed 100644
--- a/fs/namespace.c
+++ b/fs/namespace.c
@@ -139,15 +139,6 @@ static void mnt_free_id(struct mount *mnt)
 	if (unlikely(mnt->mnt.susfs_mnt_id_backup == DEFAULT_KSU_MNT_ID)) {
 		return;
 	}
-	// Now we can check if its mnt_id is sus
-	if (unlikely(mnt->mnt_id >= DEFAULT_KSU_MNT_ID)) {
-		spin_lock(&mnt_id_lock);
-		ida_remove(&mnt_id_ida, id);
-		if (mnt_id_start > id)
-			mnt_id_start = id;
-		spin_unlock(&mnt_id_lock);
-		return;
-	}
 	// Second if susfs_mnt_id_backup was set after mnt_id reorder, free it if so.
 	if (likely(mnt->mnt.susfs_mnt_id_backup)) {
 		spin_lock(&mnt_id_lock);
-- 
2.52.0

